drivers/soundwire/intel.c

```diff
@@ -126,6 +126,8 @@ struct sdw_intel {
 
 #define cdns_to_intel(_cdns) container_of(_cdns, struct sdw_intel, cdns)
 
+#define DELAY udelay_range(10, 15)
```

Comment:
Can we use usleep instead of udelay?

Reply:
@bardliao don't think you can sleep well for 10-15us. But what is `udelay_range()`? I'm not finding it and it would seem illogical to me to have a busy-waiting primitive with a range... There's `usleep_range()` and `udelay()`. Has this been compile-tested?.. Personally I'd just call `udelay(10);` explicitly everywhere, maybe only replace 10 with a macro, optionally.